### PR TITLE
socks shop port changed

### PR DESCRIPTION
**Pull request by fedya**
socks shop port changed

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/2217)

### CODE REVIEW

**Comment by chenopis** [comment]
@errordeveloper Can you double check the port number? Thx

File: docs/getting-started-guides/kubeadm.md [hunk 1]
```diff
@@ -269,7 +269,7 @@ It takes several minutes to download and start all the containers, watch the out
 
 Then go to the IP address of your cluster's master node in your browser, and specify the given port.
 So for example, `http://<master_ip>:<port>`.
-In the example above, this was `31869`, but it is a different port for you.
+In the example above, this was `30001`, but it is a different port for you.
```
Comment (errordeveloper) [inline docs/getting-started-guides/kubeadm.md]: Yes, that [is correct](https://github.com/microservices-demo/microservices-demo/blob/9cfe9cae3bbeab6d79aa68f9f6c7f842e355b555/deploy/kubernetes/complete-demo.yaml#L241), thanks for fixing this!

**Comment by errordeveloper** [comment]
Please sign the CLA. Also I am not sure why the build fails on Travis... @luxas any ideas?

**Comment by luxas** [comment]
@errordeveloper a rebase is needed

**Comment by chenopis** [comment]
@fedya Please sign the CLA when you get a chance. For expediency, I'm reimplementing this PR in #2510 to push it through to help clear our queue. Thx!

**Pull request closed without merging**